src/30_Advanced/Advanced_User_Consent_Flow.html

```diff
@@ -0,0 +1,184 @@
+        text-align: center;
+    }
+    .tabContent {
+        display: none;
+        width: 100%;
+        order: 1;
+        text-align: center;
+    }
+    .tabButton[selected]+.tabContent {
+        display: block;
+    }
+
+    amp-selector {
+      padding: 1rem;
+      margin: 1rem;
+      display: flex;
+    }
+
+  </style>
+</head>
+<body>
+
+  <!---{
+  "experiments": ["amp-consent"]
+  }--->
+
+  <!-- ## Important -->
+  <!--
+  The consent is saved to localstorage - after accepting/rejecting it once you will
+  only get the popup dialog again after deleting the localstorage content for this host.
+
+  -->
+
+  <!-- ## Basic Usage -->
+  <!--
+  The amp-consent component allows you:
+  * To ask a remote endpoint if consent is required
+  * Specify the UI to trigger the consents
+  * Specify the policy to be used for this consent
+  -->
+  <amp-consent id='amp-consent' layout='nodisplay'>
+        <script type="application/json">{
+          "consents": {
+            "DEF": {
+              "checkConsentHref": "/samples_templates/consent/getConsent",
+              "promptUI": "ui1"
+            }
+          },
+          "policy": {
+            "Policy_DEF": {
+              "waitFor": {
+                "DEF": []
+              }
+            }
+          }
+        }</script>
+        <div id="ui1">
+          <div id="consentPopup">
+             It is always good to give users the choice on data sharing.
+             Do you want to share your data with these 3rd parties?
+             <center>
+               <amp-list width="auto"
+                height="100"
+                layout="fixed-height"
+                src="/samples_templates/consent/getVendors">
+                <template type="amp-mustache">
+                  <div>
+                    {{name}}
+                  </div>
+                </template>
+              </amp-list>
+            </center>
+            <amp-selector role="tablist" layout="container" class="ampTabContainer">
+             <button role="tab" class="tabButton" option="a">YES</button>
+             <div role="tabpanel" class="tabContent">
+               Your cookie will be shared with the vendors above.
+                <br/><button on="tap:amp-consent.accept" role="button">Accept</button>
+             </div>
+             <button role="tab" class="tabButton" option="b">NO</button>
+             <div role="tabpanel" class="tabContent">
+               This site will use your cookie
+                <br/><button on="tap:amp-consent.reject" role="button">I understand</button>
+             </div>
+           </amp-selector>
+          </div>
+        </div>
+      </amp-consent>
+
+      <!--
+      You can block content until consent is given by setting the data-block-on-consent
+      attribute to any AMP component.
+      Here is a blocked image.
+      -->
+      <amp-img data-block-on-consent src="/img/landscape_lake_1280x857.jpg" width="1280" height="857" layout="responsive">
```

Comment:
It's better to give the image a fixes size (the responsive layout results in massive images on large desktops)